packages/frontend-2/pages/server-management/active-users.vue

```diff
@@ -0,0 +1,396 @@
+<template>
+  <div>
+    <Portal to="navigation">
+      <HeaderNavLink to="/server-management" name="Server Management"></HeaderNavLink>
+      <HeaderNavLink
+        to="/server-management/active-users"
+        name="Active Users"
+      ></HeaderNavLink>
+    </Portal>
+
+    <h1 class="h4 font-bold mb-4">Active Users</h1>
+
+    <FormTextInput
+      size="lg"
+      name="search"
+      :custom-icon="MagnifyingGlassIcon"
+      color="foundation"
+      full-width
+      search
+      :show-clear="!!searchString"
+      placeholder="Search Users"
+      class="rounded-md border border-outline-3"
+      @update:model-value="debounceSearchUpdate"
+      @change="(newSearchString) => searchUpdateHandler(newSearchString)"
+    />
+
+    <ServerManagementTable
+      class="mt-8"
+      :headers="[
+        { id: 'name', title: 'Name' },
+        { id: 'email', title: 'Email' },
+        { id: 'emailState', title: 'Email State' },
+        { id: 'company', title: 'Company' },
+        { id: 'role', title: 'Role' }
+      ]"
+      :items="users"
+      :buttons="[{ icon: TrashIcon, label: 'Delete', action: openUserDeleteDialog }]"
+      :column-classes="{
+        name: 'col-span-3',
+        email: 'col-span-3',
+        emailState: 'col-span-2',
+        company: 'col-span-2',
+        role: 'col-span-2'
+      }"
+      :overflow-cells="true"
+    >
+      <template #name="{ item }">
+        <div class="flex items-center gap-2">
+          <UserAvatar v-if="isUser(item)" :user="item" />
+          {{ isUser(item) ? item.name : '' }}
+        </div>
+      </template>
+
+      <template #email="{ item }">
+        {{ isUser(item) ? item.email : '' }}
```

Comment:
This column should either be truncated or wider with padding on all sides
<img width="251" alt="image" src="https://github.com/specklesystems/speckle-server/assets/938316/a7816e99-76c2-4c74-81fc-8f80f375b12a">